doc/changelog/07-commands-and-options/13096-drop-grammar-prefixes.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  Drop prefixes from grammar non-terminal names,
+  e.g. "constr:global" -> "global", "Prim.name" -> "name".
+  Visible in the output of Print * Grammar commands.
```

Comment:
```suggestion
  Visible in the output of :cmd:`Print Grammar` commands.
```

Reply:
Got it.  There's also `Print Custom Grammar`, hence the `*`.

Reply:
That was tricky! I did not get it that `*` was for an optional `Custom`.